index.js

```diff
@@ -523,6 +546,11 @@
       this === other;
   }
 
+  //  Boolean$prototype$lte :: Boolean ~> Boolean -> Boolean
+  function Boolean$prototype$lte(other) {
+    return typeof other === typeof this && this <= other;
+  }
+
```

Comment:
`typeof other === typeof this` is now a given as a result of #47. Let's use this approach:

```javascript
//  Boolean$prototype$lte :: Boolean ~> Boolean -> Boolean
function Boolean$prototype$lte(other) {
  return typeof this === 'object' ?
    lte(this.valueOf(), other.valueOf()) :
    this <= other;
```

I'm not happy with the type coercion in `this <= other`. I believe `!this || other` is equivalent.